cmd/utils/tps/tps.go

```diff
@@ -0,0 +1,108 @@
+// This Source Code Form is subject to the terms of the MIT License.
+// If a copy of the MIT License was not distributed with this
+// file, you can obtain one at https://opensource.org/licenses/MIT.
+//
+// Copyright (c) DUSK NETWORK. All rights reserved.
+
+package tps
+
+import (
+	"context"
+	"errors"
+	"time"
+
+	"github.com/dusk-network/dusk-protobuf/autogen/go/node"
+	"github.com/sirupsen/logrus"
+	"google.golang.org/grpc"
+)
+
+const (
+	retryAmount   = 5
+	retryDelay    = 5 * time.Second
+	defaultAmount = 100
+)
+
+// StartSpamming transactions from the given GRPC address. The utility will spam
+// Transfer transactions right back to the wallet it's coming from.
+func StartSpamming(addr string, delay int, amount uint64) error {
+	if addr == "" {
+		return errors.New("no GRPC address provided")
+	}
+
+	if amount == 0 {
+		amount = defaultAmount
+	}
+
+	conn, err := grpc.Dial(addr, grpc.WithInsecure())
```

Comment:
Could we add here `DialTimeout`  support? 
Also, we might need `grpc.WithBlock()` option.

Reply:
Good catch, will add!